### PR TITLE
when searching via q allow archived results to be returned

### PR DESCRIPTION
**Pull request by samjacobclift**
**headlines**

1) Show archived results when filtering by a query param

Not sure this entirely the best approach wdyt?

related issue https://github.com/octobox/octobox/pull/361

### CODE REVIEW

File: app/controllers/notifications_controller.rb [hunk 1]
```diff
@@ -200,6 +200,8 @@ def current_notifications(scope = notifications_for_presentation)
       scope = scope.send(sub_scope, params[sub_scope]) if params[sub_scope].present?
     end
     scope = scope.search_by_subject_title(params[:q])   if params[:q].present?
+    scope = scope.unscope(where: :archived)  if params[:q].present?
```
Comment (tarebyte) [inline app/controllers/notifications_controller.rb]: My helping my inner rubocop and aligning these statements?

```diff
     scope = scope.search_by_subject_title(params[:q])   if params[:q].present?
+    scope = scope.unscope(where: :archived)             if params[:q].present?
```

File: test/controllers/notifications_controller_test.rb [hunk 2]
```diff
@@ -57,6 +57,15 @@ class NotificationsControllerTest < ActionDispatch::IntegrationTest
...
 
+  test 'shows archived search results by default' do
+    sign_in_as(@user)
+    5.times.each { create(:notification, user: @user, archived: true, subject_title:'release-1') }
+
+    get '/?q=release'
+    assert_equal assigns(:notifications).length, 5
+  end
+
+
```
Comment (tarebyte) [inline test/controllers/notifications_controller_test.rb]: Please remove the extra line here

**Comment by samjacobclift** [comment]
@tarebyte sure, let me clean up those nits!

**Comment by andrew** [comment]
👍  do we have/need a way to visually show the difference between notifications in the inbox vs archived when they are listed together?

**Comment by andrew** [comment]
Not sure why travis hasn't kicked off a test here, @samjacobclift can you rebase your branch and force push it to get travis to run the tests again?

**Comment by samjacobclift** [comment]
sure will do now

**Comment by chrisarcand** [comment]
> perhaps I should just squish the commits

🙇

**Comment by samjacobclift** [comment]
@chrisarcand done

**Comment by andrew** [comment]
Very strange, still no travis, going to try turning it off and on again

**Comment by andrew** [comment]
🚢

**Pull request merged**